framework/lib/components/createable-select-dropdown/types.ts

```diff
@@ -51,4 +51,9 @@ export interface CreatableSelectDropdownProps {
    *
   */
   variant?: Variant
+
+  /**
+   * The currently selected option. Useful for initializing the component with an option selected.
```

Comment:
`currently selected option` indicates that the property can control the current state of the select. That is not true.

As @TheLudd pointed out, this should be called `initialValue`